mex/extractors/biospecimen/main.py

```diff
@@ -47,7 +47,7 @@ def biospecimen_resources() -> list[BiospecimenResource]:
 
 
 @asset(group_name="biospecimen")
-def extracted_mex_persons(
+def biospecimen_extracted_mex_persons(
```

Comment:
why do we need to keep the `mex` in there?

```suggestion
def biospecimen_extracted_persons(
```